### ISSUE TITLE
Option for disabling (or simplifying) status bar output

### ISSUE THREAD

**Issue by TheOnlyRew** [SublimeLinter/SublimeLinter3#338]
tl;dr -- I don't like that SublimeLinter always prints the error text of the line the cursor is resting on to the status bar at the bottom of the window. I'd appreciate a setting that would allow for some control of this feature.

_Longer Version_
I hate clutter. I love SublimeLinter-annotations though, especially in files for which I'm not using more complex linters. When my cursor hits a line with a `TODO` or a `FIXME`, the entire text of that line (which Annotations considers an error) is pasted into the status bar, even though I can clearly read the line that my cursor is resting on. Like I said, I hate clutter.

I'd appreciate a setting -- preferably a boolean so that `toggle_setting` could be used with it -- that would allow me to control whether or not the text of the error gets displayed in the status bar. Perhaps something like...

```
{
    "user": {
        "status_bar_error_text": false,
        . . .
    }
}
```

(Edit: `full_error_output` was a terrible name. Updated.)

The salient code lives at
https://github.com/SublimeLinter/SublimeLinter3/blob/master/sublimelinter.py#L383-L403
Most specifically the `status += '; '.join(line_errors)` on line 401.

**Comment by braver** [comment]
See #618
